docs/api/global_context.rst

```diff
@@ -0,0 +1,33 @@
+.. _global_context:
+
+Global Context
+==============
+
+If you have a single-threaded application that generates many objects,
+enabling the use of the global context can provide performance enhancements.
+
+.. warning:: The global context is not thread safe.
+.. warning:: The global context does not autoclose the database.
```

Comment:
Reading this I wonder: should I then do this manually after I am done using the context, and how can I do this?

Reply:
Good call out. I should re-word that for clarity. The connection to the database persists through the duration of the program and is closed once the program terminates.